src/Logger/BaseTextFileLogger.php

```diff
@@ -43,13 +43,11 @@
 use Infection\Mutant\MutantExecutionResult;
 use const PHP_EOL;
 use function sprintf;
-use function str_repeat;
-use function strlen;
 
 /**
  * @internal
  */
-final readonly class TextFileLogger implements LineMutationTestingResultsLogger
```

Comment:
extracted a new `BaseTextFileLogger` out of `TextFileLogger` so I could easily overwrite the markdown headline formatting and insert the github action log section wrappers in `GitHubActionsLogTextFileLogger`

no functional changes to `TextFileLogger` (which is now re-named to `MarkdownTextFileLogger`)